docs/src/app/(public)/(content)/react/overview/quick-start/page.mdx

```diff
@@ -0,0 +1,47 @@
+# Quick Start
+
+<Subtitle>A quick guide to getting started with Base UI.</Subtitle>
+<Meta name="description" content="A quick guide to getting started with Base UI." />
+
+## Installation
+
+Install Base UI using a package manager. All components are included in this single package.
+
+```
+npm i @base-ui-components/react
+```
+
+## Assembling a component
+
+Import a component, assemble its parts, and apply styles. This example uses Tailwind, but you can use CSS Modules, CSS-in-JS, plain CSS, or any styling solution you prefer.
+
+```
+import * as React from 'react';
+import { Popover } from '@base-ui-components/react/popover';
+
+export default function MyPopover() {
+  return (
+    <Popover.Root>
+      <Popover.Trigger className="flex size-10 items-center justify-center rounded-md border border-gray-200 bg-gray-50 text-gray-900 select-none hover:bg-gray-100 focus-visible:outline-2 focus-visible:-outline-offset-1 focus-visible:outline-blue-800 active:bg-gray-100 data-[popup-open]:bg-gray-100">
+        Notifications
+      </Popover.Trigger>
+      <Popover.Portal>
+        <Popover.Positioner sideOffset={8}>
+          <Popover.Popup className="origin-[var(--transform-origin)] rounded-lg bg-[canvas] px-6 py-4 text-gray-900 shadow-lg shadow-gray-200 outline outline-gray-200 transition-[transform,scale,opacity] dark:shadow-none dark:-outline-offset-1 dark:outline-gray-300 [[data-starting-style],[data-ending-style]]:scale-90 [[data-starting-style],[data-ending-style]]:opacity-0">
+            <Popover.Title className="text-base font-medium">
+              Notifications
+            </Popover.Title>
+            <Popover.Description className="text-base text-gray-600">
+              You are all caught up. Good job!
+            </Popover.Description>
+          </Popover.Popup>
+        </Popover.Positioner>
+      </Popover.Portal>
+    </Popover.Root>
+  );
+}
+```
+
+## Next steps
+
+This walkthrough outlines the basics of putting together a Popover component with Base UI. Check out the Popover component documentation to learn how to configure it further. For example, you might want to add the `Popover.Arrow` part, or configure some props.
```

Comment:
Maybe we can rephrase the Next steps to point people to browser the available components instead of the Popover page. What do you think?

Reply:
One other thing we can add as a semi-optional step is to add `isolation: isolate` to your root layout in order for portals to layer automatically (this is the new `position: relative; z-index: 0`)

Reply:
> point people to browser the available components

I agree this would be better, but we don't have a page listing available components, and won't have time to create one.

> using a simple component that’s familiar to everyone
> Using a simple component on the Quick start makes sense
I stuck with Popover because I don't think using a simple component like a button is a good idea. When exploring a new headless UI lib, it's not helpful or exciting to learn how to style a button. I think it's more exciting and educational to jump into a popup component. It's also useful to learn some Base UI concepts like composition and some APIs. I chose Popover because I think it's the simplest component that's interesting.

> One other thing we can add as a semi-optional step is to add isolation: isolate

I added this, not sure if I demonstrated it in the most useful way?

Reply:
> One other thing we can add as a semi-optional step is to add isolation: isolate to your root layout in order for portals to layer automatically (this is the new position: relative; z-index: 0)

Curious what this achieves? Also in the PR it was made to `<html>`, did you mean the root app div where the portals go outside?